### ISSUE TITLE
Expansion while registering

### ISSUE THREAD

**Issue by sndcode** [paintballrefjosh/MaNGOSWebV4#85]
Either im blind or is the site config menu in admin panel missing the expansion setting that sets the highest available expansion users can register with

**Comment by paintballrefjosh** [comment]
I don't believe this is an option in MWE, did you have a reference to it somewhere else?
